modules/core/arrow-generic/build.gradle

```diff
@@ -0,0 +1,18 @@
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlinVersion"
```

Comment:
shouldn't we move on to newer gradle versions and use `implementation` (for non allowing transitive deps) and `api` (for allowing transitive deps) ? these already old `compile` statements are throwing warnings at compile time on projects compiled with newer versions of gradle (or the wrapper).

Reply:
Sounds good, but since all modules are coded now like that it should be done as a separate issue being unrelated to Generic.

Reply:
Nevermind my comment, that's part of the android gradle plugin, not part of gradle itself.